content/applications/inventory_and_mrp/purchase/products/temporary_reordering.rst

```diff
@@ -0,0 +1,120 @@
+==========================
+Temporary reordering rules
+==========================
+
+Some businesses require certain products to always have a minimum quantity of stock on-hand at any
+given time. To avoid stock falling below a certain threshold, companies can create *reordering
+rules* in Odoo to automate purchase orders for specific products.
+
+Reordering rules keep the forecasted stock levels above a certain threshold without exceeding a
+specified upper limit. When a product has a reordering rule applied to it, and that product's
+quantity falls below a specified number, Odoo uses the specified *route* on the product form
+(e.g. *Buy* or *Manufacture*), and creates an order to replenish the stock.
+
+However, there are some instances in which businesses might not want or need specific products to be
+replenished automatically. In such cases, they can rely on "temporary" reordering rules.
+
+In Odoo, when a product has 0 quantity in stock, has a *Buy* route selected, and is included in a
+sales order (SO), Odoo creates a "temporary" reordering rule in the replenishment dashboard for that
+product, which is deleted upon confirmation of the purchase order (PO) created for that product.
```

Comment:
Optional rewording suggestion to move all the conditions to trigger the reordering rule in a list since I personally found it a little difficult to keep track of them all. Feel free to reword this more!
```suggestion
In Odoo, a "temporary" reordering rule is created in the replenishment dashboard when a product:

#. with a *Buy* route
#. has no reordering rule configured
#. has 0 quantity in stock
#. is included in a sales order (SO).

This rule is deleted upon confirmation of the purchase order (PO) generated for the product.
```